audio/mixer_intern.h

```diff
@@ -81,7 +82,7 @@ class MixerImpl : public Mixer {
 
 public:
 
-	MixerImpl(uint sampleRate);
+	MixerImpl(uint sampleRate, uint outBufSize = -1);
```

Comment:
assign -1 to uint is good idea. 0 is not enough?

Reply:
Whoops, it should have been an int, not a unit.
But I suppose you are right, I can't think of a reason why -1 might be better than 0 as a special value.